components/com_fabrik/models/list.php

```diff
@@ -2320,12 +2320,23 @@ public function _addLink($data, &$elementModel, $row, $repeatCounter = 0)
 		$rowId = $this->getSlug($row);
 		$isAjax = $this->isAjaxLinks() ? '1' : '0';
 		$isCustom = $customLink !== '';
+		$isIcon = FALSE;
+		
+		/* Check if the data component is html, if it is an icon lnk and image parse it out */
+		if (class_exists('DOMDocument')) {
+			$html = new DOMDocument;
+			$html->loadHTML($data);
+			$a = $html->getElementsByTagName('a')->item(0);
+			$img = $html->getElementsByTagName('img')->item(0);
+			if (isset($html) && isset($a) && isset($img))
+				$isIcon = TRUE;
```

Comment:
Put {} around the body.  I know it's not technically required for a single statement.